src/elements/content-uploader/__tests__/ContentUploader.test.js

```diff
@@ -579,4 +579,32 @@ describe('elements/content-uploader/ContentUploader', () => {
             expectAutoExpandStateToBe(false);
         });
     });
+
+    describe('componentDidMount()', () => {
+        it('adds files to upload queue if allowPrepopulateFiles is true and files are provided', () => {
+            const files = createMockFiles(3);
+            const wrapper = getWrapper({
+                allowPrepopulateFiles: true,
+                files,
+            });
+            const instance = wrapper.instance();
+            instance.addFilesToUploadQueue = jest.fn();
+
+            instance.componentDidMount();
+            // Assert that addFilesToUploadQueue is called with the correct arguments
+            expect(instance.addFilesToUploadQueue).toHaveBeenCalled();
```

Comment:
nit: this repo prefers present tense aliases of these jest functions:
```suggestion
            expect(instance.addFilesToUploadQueue).toBeCalled();
```